runtime/browser/ui/native_app_window_views.cc

```diff
@@ -229,7 +231,17 @@ gfx::ImageSkia NativeAppWindowViews::GetWindowAppIcon() {
 }
 
 gfx::ImageSkia NativeAppWindowViews::GetWindowIcon() {
-  return *icon_.ToImageSkia();
+  const gfx::ImageSkia *result =  icon_.ToImageSkia();
+
+  const std::vector<float> oldSupportedScales =
+      gfx::ImageSkia::GetSupportedScales();
+  std::vector<float> scale = { 1.0f };
+  gfx::ImageSkia::SetSupportedScales(scale);
+  // Ensure at least the ImageSkiaRep for the default 1x scale is attached to
+  // ImageSkia's storage
+  result->EnsureRepsForSupportedScales();
```

Comment:
The function ToImageSkia always return a valid pointer, so let's keep the original no null-checked.